pages/blog/_posts/open communication.md

```diff
@@ -0,0 +1,38 @@
+# What are your weekend plans?
+
+## Open communication as a tool to break down barriers   
+ 
+Do you know what your coworkers are doing this weekend, and should you know? Do you share your day, evening, and weekend plans in your work calendar or to your colleagues? From my experience, having an open work calendar and sharing personal information can drastically improve organizational culture.  
+
+## We are all just humans  
+Every one of my coworkers are humans in different stages of life. Some are fresh out of college, some are getting married, some are establishing a family, and others getting a dog. But how much do you actually know about your coworkers, and do you take the time to get to know them? 
+
+## Going to the doctor
+We openly share non-work related information on the same level as work-related information. Everything from doctor appointments, that we will run late for office, that we deliver or pick up kids in kindergarten, veterinarian appointments, and we add our sports games and events in our calendar. 
+
+You might think this is information-overload in an already hectic day. Nonetheless, it provides opportunities. Opportunities to see what is happening in my co-worker's life and we connect on a personal level. It provides possibilities to follow up about their sports games, and asks how their kids are doing. 
+
+## I do not want to be perceived as a “Mom”
```

Comment:
This is a quote, so maybe it should be `"I do not want to be perceived as a 'Mom'"`